recipes/libgsasl/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "1.8.0" %}
+
+package:
+  name: libgsasl
+  version: "{{ version }}"
+
+source:
+  fn: libgsasl-{{ version }}.tar.gz
+  url: ftp://ftp.gnu.org/gnu/gsasl/libgsasl-{{ version }}.tar.gz
+  md5: 5dbdf859f6e60e05813370e2b193b92b
+
+build:
+  number: 0
+  skip: True  # [not linux]
```

Comment:
same as hdfs3 - please add a comment as to why this is linux-only, and what it might take to get it building elsewhere.
 Add a line note